### ISSUE TITLE
Add new 'README.rst' file in ZOO-Project source

### ISSUE THREAD

**Issue by jmckenna** [ZOO-Project/migration#8]
- placed in root directory
- will then display nicely below the source code on the page: https://github.com/ZOO-Project/ZOO-Project
- example README.rst : https://github.com/MapServer/MapServer/blob/main/README.rst

**Comment by jmckenna** [comment]
Note that this file should also include the license text.

**Comment by gfenoy** [comment]
I have just created a tentative README.md file available from [here](https://github.com/ZOO-Project/ZOO-Project/blob/main/README.md).

Any feedback welcome.

**Comment by venka-foss4g** [comment]
Maybe mention gdal, cgal in the README.md in addition to Orfeo ToolBox
and SAGA-GIS

**Comment by gfenoy** [comment]
Thanks a lot for the feedbacks.

As I was not very sure where to place it, I added the reference to GDAL and CGAL within the ZOO-Services description.

With this comment, I noted also that there were no reference to the MapServer software. Also, now that I have mentioned MapServer, I think that GDAL should also be mentioned in the ZOO-Kernel section as it is used to extract metadata informations that are required to publish the data to MapServer.

Please, provide comment on the part trying to mention GDAL, CGAL and, trying to avoid too much acronym on the description page even if there is already a lot.

**Comment by venka-foss4g** [comment]
I agree with avoiding too many acronyms. Maybe some sentence like below will suffice

"ZOO-Kernel exploits the Raster and Vector drivers supported by GDAL/OGR to read/write geospatial data in commonly used data formats. ZOO-Kernel provides access to GDAL/OGR utilities and geometric algorithms available in CGAL are also implemented as WPS. Other open source geospatial can be easily integrated and we do it as an when the requirement and funding is available."

**Comment by gfenoy** [comment]
Thanks for the feedbacks @venka-foss4g. Nevertheless, I am not sure where to put such a sentence. I tried to keep ZOO-Services and ZOO-Kernel in distinct sections.
